src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -425,15 +444,16 @@ class GroupReadsByUmi
   @arg(flag='T', doc="The output tag for UMI grouping.") val assignTag: String = "MI",
   @arg(flag='m', doc="Minimum mapping quality.")         val minMapQ: Int      = 30,
```

Comment:
Changed both the doc and default value:
```suggestion
  @arg(flag='m', doc="Minimum mapping quality for mapped reads.")         val minMapQ: Int      = 1,
```